### PR TITLE
DW-4676: Add Rstudio Repository

### PR DESCRIPTION
**Pull request by caseyr232**

### CODE REVIEW

File: docker-rstudio-oss.tf [hunk 1]
```diff
@@ -0,0 +1,76 @@
+resource "github_repository" "docker-rstudio-oss" {
+  name        = "docker-rstudio-oss"
+  description = "A JupyterHub container with required extensions and libraries"
```
Comment (liamraemclauchlan) [inline docker-rstudio-oss.tf:3]: Accurate?

File: docker-rstudio-oss.tf [hunk 2]
```diff
@@ -0,0 +1,76 @@
+resource "github_repository" "docker-rstudio-oss" {
```
Comment (dan-hill2802) [inline docker-rstudio-oss.tf:1]: can you make all Terraform resource names use underscores please

File: docker-rstudio-oss.tf [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  plaintext_value = "${var.dockerhub_username}"
+}
+
+resource "github_actions_secret" "docker-rstudio-oss-snyk-token" {
+  repository      = "${github_repository.docker-rstudio-oss.name}"
+  secret_name     = "SNYK_TOKEN"
+  plaintext_value = "${var.snyk_token}"
+}
+
+resource "github_actions_secret" "docker-rstudio-oss-slack-webhook" {
```
Comment (dan-hill2802) [inline docker-rstudio-oss.tf:72]: are you planning on using Slack notification in the pipeline?

File: docker-rstudio-oss.tf [hunk 4]
```diff
@@ -0,0 +1,76 @@
+resource "github_repository" "docker-rstudio-oss" {
+  name        = "docker-rstudio-oss"
+  description = "A JupyterHub container with required extensions and libraries"
+
+  allow_merge_commit     = false
+  delete_branch_on_merge = true
+  auto_init              = true
+  has_issues             = false
+
```
Comment (dan-hill2802) [inline docker-rstudio-oss.tf:9]: there should be a template block here. The readme says to use https://github.com/dwp/dataworks-github-config/blob/master/docker-template-repository.tf.sample as your starting point

**Pull request merged**